lib/vssolution.cpp

```diff
@@ -2351,21 +2350,36 @@ void VisualizationSceneSolution::Draw()
       Set_Transparency();
    }
 
+   if (GetUseTexture())
+   {
+      glEnable (GL_TEXTURE_1D);
+      glColor4d(1, 1, 1, 1); // default color
+   }
+
    // draw elements
    if (drawelems)
    {
-      if (GetUseTexture())
+      glCallList(displlist);
+   }
+
+   // draw ordering -- color modes
+   if (draworder)
+   {
```

Comment:
This `if` is redundant. The same is true for the black&white mode section below and in the 3D version.

Reply:
Fixed in f96a475